js/views/UserCard.js

```diff
@@ -73,19 +72,16 @@ export default class extends BaseVw {
   }
 
   loadUser(guid = this.guid) {
-    let profile;
     this.fetched = true;
 
     if (guid === app.profile.id) {
-      // don't fetch our this user's own profile, since we have it already
-      this.profileFetch = $.Deferred().resolve();
-      profile = app.profile;
+      // don't fetch this user's own profile, since we have it already
+      this.profileFetch = $.Deferred().resolve(app.profile);
     } else {
-      profile = new Profile({ peerID: guid });
-      this.profileFetch = profile.fetch();
+      this.profileFetch = getCachedProfiles([guid])[0];
```

Comment:
I decided to use the cached profiles after all. The latency improvement is substantial. And... even having the freshest moderator info doesn't really solve the issue. After a mod is added, whether it's 1 minute, 1 hour or 6 months later, that mod can change their terms and the users using that mod would have no idea. Perhaps it's as simple as sending a notification to users of a mod if the mod changes their terms? Seems like a bigger UX discussion we could circle back to.